app/src/db/migrations/20241028000000_012-user-contacts.ts

```diff
@@ -0,0 +1,266 @@
+/*
+ * Two new tables are created:
+ *    contact
+ *    activity_contact
+ *
+ * Contact information from submission & enquiry is moved to these tables and then dropped from originals
+ */
+
+/* eslint-disable max-len */
+import { v4 as uuidv4 } from 'uuid';
+
+import stamps from '../stamps';
+
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return (
+    Promise.resolve()
+
+      // Create public schema tables
+      .then(() =>
+        knex.schema.createTable('contact', (table) => {
+          table.uuid('contact_id').primary();
+          table
+            .uuid('user_id')
+            .nullable()
+            .references('user_id')
+            .inTable('user')
+            .onUpdate('CASCADE')
+            .onDelete('CASCADE');
+          table.text('first_name');
+          table.text('last_name');
+          table.text('phone_number');
+          table.text('email');
```

Comment:
Business question: We should ideally have some form of compound uniqueness constraint over first_name, last_name and email, but this will call into question if or how we'd be able to de-duplicate information. We'll likely have to leave it as is due to uncertainty, but this can become a data hazard if left unconstrained for too long.